CameraOverlay/overlay_all_stats.py

```diff
@@ -28,12 +29,12 @@ def on_connect(self, client, userdata, flags, rc):
 		topics_qos = list(zip(topic_values, at_most_once_qos))
 		client.subscribe(topics_qos)
 
-
 		# Add static text
 		self.base_canvas.draw_text("REC Power:", (5, self.text_height * 1))
 		self.base_canvas.draw_text("Power:", (5, self.text_height * 2))
 		self.base_canvas.draw_text("Cadence:", (5, self.text_height * 3))
 		self.base_canvas.draw_text("Distance:", (5, self.text_height * 4))
+		self.base_canvas.draw_text("Message:",(5,self.text_height * 5 ),size = 1)
```

Comment:
Might wanna add a space after each `,`. Be interesting to see how your changes go through `pylint`.